keycode.cabal

```diff
@@ -37,7 +37,7 @@ library
                      , containers
                      , ghc-prim
   if impl(ghc >= 8.0)
-    build-depends:     template-haskell >= 2.11 && < 2.13
+    build-depends:     template-haskell >= 2.11.1.0 && < 2.13
```

Comment:
@RyanGlScott no problem, thanks. Decline PR then, I'll keep it in my fork until I find a solution to keep 2.11.0.0